scripts-available/CDB_OverviewsSupport.sql

```diff
@@ -0,0 +1,173 @@
+-- Auxiliary overviews FUNCTIONS
```

Comment:
Renaming a file for a bugfix really obscures the review. Have you actually modified anything in this file